build.gradle

```diff
@@ -1,115 +1,94 @@
 plugins {
-    id 'com.google.protobuf' version '0.8.8' apply false
-    id 'com.palantir.docker' version '0.25.0' apply false
-    id "de.undercouch.download" version "4.0.4" apply false
-    id 'org.jetbrains.kotlin.jvm' version '1.3.72' apply false
-    id "com.jfrog.artifactory" version '4.15.1'
+    id 'com.palantir.docker' version '0.25.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
 }
 
-allprojects {
-    apply plugin: 'base'
-    apply plugin: 'maven-publish'
-    apply plugin: 'java-library'
+apply plugin: 'maven-publish'
+apply plugin: 'application'
+apply plugin: 'kotlin'
```

Comment:
Using apply is an old way of applying plugins. Gradle recommends using the new way
https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application